models/review.js

```diff
@@ -0,0 +1,14 @@
+const db = require("../db/connection");
+
+
+exports.selectReview = (id) => {
+    return db.query(`SELECT * FROM reviews WHERE ${id} = review_id;`)
+        .then((data) => {
+            if (data.rows.length === 0) {
+                return Promise.reject({ status: 404, message: 'not found'})
+               }
+            return data.rows
```

Comment:
Here you could return data.rows[0] so that you don't have an array of one object in your response body.